packages/gatsby-plugin-google-analytics/src/__tests__/gatsby-browser.js

```diff
@@ -72,6 +75,15 @@ describe(`gatsby-plugin-google-analytics`, () => {
           expect(setTimeout).toHaveBeenCalledTimes(1)
           expect(window.ga).toHaveBeenCalledTimes(2)
         })
+
+        it(`uses setTimeout when pageTransitionDelay is set`, () => {
+          onRouteUpdate({}, { pageTransitionDelay: 1000 })
+
+          jest.runAllTimers()
+
+          expect(setTimeout).toHaveBeenCalledTimes(1)
```

Comment:
We should check if the correct delay is used
```suggestion
          expect(setTimeout).toHaveBeenCalledWith(expect.any(Function), 1000)
```